### DISCUSSION TITLE
Glossary: What is "key"?

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10071] [Q&A]
![image](https://github.com/WeblateOrg/weblate/assets/11861496/4d6f2341-8bfb-469d-a796-39a18482b8c7)

In Glossary there is a field "key". I don't know what it is for and if I need to use it or not.

https://translate.codeberg.org/browse/backintime/glossary/en/?q=

**Comment by Geeyun-JY3** [comment]
https://docs.weblate.org/en/weblate-5.0.2/formats.html#context

**Comment by buhtz** [comment]
> https://docs.weblate.org/en/weblate-5.0.2/formats.html#context

Thank you for that link. But it doesn't help me to unterstand. Can you 
give me an example?

**Comment by nijel** [comment]
It is used to differentiate the strings when there are multiple strings with the same source.
